src/canvas/StaticCanvas.ts

```diff
@@ -322,15 +347,21 @@ export class StaticCanvas<
    * @param {Boolean}       [options.backstoreOnly=false] Set the given dimensions only as canvas backstore dimensions
    * @param {Boolean}       [options.cssOnly=false]       Set the given dimensions only as css dimensions
    */
+  setDimensions(
+    dimensions: Partial<CSSDimensions>,
+    options?: { cssOnly?: true; backstoreOnly?: false }
+  ): void;
   setDimensions(
     dimensions: Partial<TSize>,
-    { cssOnly = false, backstoreOnly = false }: TCanvasSizeOptions = {}
+    options?: { backstoreOnly?: true; cssOnly?: false }
+  ): void;
+  setDimensions(dimensions: Partial<TSize>, options?: never): void;
+  setDimensions(
+    dimensions: Partial<TSize | CSSDimensions>,
+    options?: TCanvasSizeOptions
   ) {
-    this._setDimensionsImpl(dimensions, {
-      cssOnly,
-      backstoreOnly,
-    });
-    if (!cssOnly) {
+    this._setDimensionsImpl(dimensions, options);
+    if (options && !options.cssOnly) {
```

Comment:
@asturur  this is a regressions
if no options are passed the canvas does not render

Reply:
Quoting: A typings PR should be only a typings PR

Reply:
@ShaMan123 you are using snarky comments since yesterday.
I don't like that.
This PR has been extensively looked by me and you and a bug slipped in.
It was a typing pr indeed, no logic was meant to change

Reply:
And the PR is titled as fix() and includes the word typings is not that i wrote `chore(TS)` and changed the code